recipe/meta.yaml

```diff
@@ -1,80 +1,92 @@
 {% set name = "pyxrf" %}
 {% set version = "1.0.2" %}
+{% set sha256 = "0e4c000392e70ec04aa38492f8a4868532eae4dfc09e965d1eb5f389f48c94c9" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 0e4c000392e70ec04aa38492f8a4868532eae4dfc09e965d1eb5f389f48c94c9
+  sha256: {{ sha256 }}
 
 build:
   noarch: python
-  number: 0
+  number: 1
   entry_points:
-    - pyxrf = pyxrf.gui:run
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
+    - pyxrf = pyxrf.pyxrf_run:run
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
-    - pip
     - python >=3.6
-    - setuptools
+    - pip
   run:
     - python >=3.6
-    - atom =0.4.3
+    - atom ==0.4.3
     - dask
     - distributed
+    - h5py >=2.9.0
+    - ipywidgets
     - jsonschema
-    - h5py
-    - lmfit
-    - matplotlib
+    - jupyter
+    - lmfit >=0.8.3
+    - matplotlib <=3.2.2
```

Comment:
Why is this upper constraint on matplotlib necessary?

Reply:
@dmgav discovered some segfaults with 3.3.
@dmgav, @tacaswell, could you please provide more details?

Reply:
PyXRF segfaults with MPL 3.3. We are investigating the issue. It is better to put upper limit than instruct users to downgrade manually.